db/schema.rb

```diff
@@ -573,6 +574,8 @@
     t.datetime "default_currency_confirmed_at"
     t.datetime "member_at"
     t.datetime "send_emails", default: -> { "CURRENT_TIMESTAMP" }
+    t.text "card_id"
+    t.index ["card_id"], name: "index_uphold_connections_on_card_id"
```

Comment:
I don't necessarily think we need create a `card_id` field because it's equivalent to the existing `address` field.

Reply:
`address` isn't the same as `card_id` from what I can tell